lib/msf/core/exploit/remote/smb/server/hash_capture.rb

```diff
@@ -103,9 +95,31 @@ def report_ntlm_type3(address:, ntlm_type1:, ntlm_type2:, ntlm_type3:)
       # found_host.os_name = credential_options[:client_os_version]
       # found_host.save!
 
+      search_options = {
+        ptype: credential_options[:jtr_format],
```

Comment:
What is the reasoning to have `pytpe` and `jtr_format` both reference the same value?